react/Listbox.tsx

```diff
@@ -40,7 +44,8 @@ export const ListboxButton: React.FC<ButtonProps & { className?: string }> = ({
       className={classNames(
         styles.button,
         className,
-        'h-100 flex items-center bg-base c-on-base ba bw1 br2 b--muted-4 hover-b--muted-3 pv4 pv3-ns pl4 pr5 outline-0 pointer'
+        'h-100 flex items-center bg-base c-on-base pv4 pv3-ns pl4 pr5 outline-0 pointer',
+        { 'ba bw1 br2 b--muted-4 hover-b--muted-3 ': !plain }
```

Comment:
You don't need this extra space after `hover-b--muted-3`, do you?

Reply:
no, I don't!